src/gainmap.c

```diff
@@ -660,6 +655,22 @@ avifResult avifRGBImageComputeGainMap(const avifRGBImage * baseRgbImage,
         }
     }
 
+    // Populate the gain map metadata's headrooms.
+    gainMapMetadata.baseHdrHeadroom = log2f(AVIF_MAX(baseMax, kEpsilon));
+    gainMapMetadata.alternateHdrHeadroom = log2f(AVIF_MAX(altMax, kEpsilon));
+
+    // The gainmap is the log-ratio of the higher-headroom (more HDR) image to the lower-headroom
+    // (less HDR) image. Reverse the gainmap if this is the case.
```

Comment:
It is not clear what "this is the case" means. Could you clarify that? Should we say "if this is **not** the case"?